python/exercise1/vowel_counter.py

```diff
@@ -2,17 +2,19 @@
 
 def num_vowels(text):
     """Return the number of vowels in string."""
-    vowels = "aeiou"
+    vowels = "aeiouy"
     num = 0
     for v in vowels:
         num += text.lower().count(v)        
     return num
 
 def num_consonants(text):
-    vowels = "aeiou"
+    vowels = "aeiouy"
+    num = 0
     for letter in text:
         if letter not in vowels:
-            print("consonant", letter)
+           num += text.lower().count(letter)
```

Comment:
perhaps move .lower() to line 14 so it works on the vowels also

Also, should handle numbers & other characters separately. Currently, &, I, and 2 would all be counted by this function, though they are not consonants